lib/game.rb

```diff
@@ -0,0 +1,17 @@
+class Game
+  attr_reader :name, :cost, :national_drawing
+
+  def initialize(name, cost, national_drawing)
+    @name             = name
+    @cost             = cost
+    @national_drawing = true
```

Comment:
Similar to the `out_of_state?` method, a Game will not always be a national drawing.  Based on the interaction pattern, we should be able to create a Game with either 2 or 3 arguments, with the 3rd argument indicating if it is or is not a national game:

```ruby
def initialize(name, cost, national_drawing = false)
  ...
  @national_drawing = national_drawing
end
```